### PR TITLE
Refactor hidapi submodule

### PR DESCRIPTION
**Pull request by p01arst0rm**
moved hidapi submodule into its own directory structure

### CODE REVIEW

File: 3rdparty/hidapi/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+	add_subdirectory(hidapi/windows EXCLUDE_FROM_ALL)
+	target_include_directories(hidapi-hid PUBLIC hidapi/hidapi)
+
+	target_link_libraries(3rdparty_hidapi INTERFACE hidapi-hid Shlwapi.lib)
+else()
+	add_subdirectory(hidapi/libusb EXCLUDE_FROM_ALL)
+	target_include_directories(hidapi-libusb PUBLIC hidapi/hidapi)
+
+	target_link_libraries(3rdparty_hidapi INTERFACE hidapi-libusb usb)
+endif()
```
Comment (Megamouse) [inline 3rdparty/hidapi/CMakeLists.txt:26]: missing newline

Comment (p01arst0rm) [reply 3rdparty/hidapi/CMakeLists.txt:26]: oops :)

**Comment by Megamouse** [comment]
@hcorion please review. Let's keep the tradition shall we? 😆

**Review by hcorion — COMMENTED** [review]
LGTM 👍 Pretty straightforward

**Review by hcorion — APPROVED** [review]

**Comment by Nekotekina** [comment]
Don't forget to squash merge.

**Pull request merged**